crates/op-rbuilder/src/args/op.rs

```diff
@@ -93,11 +93,19 @@ pub struct FlashblocksArgs {
     )]
     pub flashblocks_addr: String,
 
-    /// flashblock block time in milliseconds
+    /// Number of Flashblocks per block
     #[arg(
-        long = "flashblocks.block-time",
-        default_value = "250",
-        env = "FLASHBLOCK_BLOCK_TIME"
+        long = "flashblocks.per-block",
+        default_value = "10",
+        env = "FLASHBLOCKS_PER_BLOCK"
     )]
-    pub flashblocks_block_time: u64,
+    pub flashblocks_per_block: u64,
+
+    /// Overhead for reset of block production (e.g. new payload request to sequencer EL)
+    #[arg(
+        long = "flashblocks.block-overhead",
```

Comment:
this is a constant offset but I'd imagine the overhead latency varying considerable block to block. Is there a way to calculate some offset based on block time and how close the block building starts from the end of the slot?